packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -49,24 +51,52 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
     @state()
     private _isModalOpen = false;
 
+    @state()
+    private locale: CookieBannerLocale = defaultLocale;
```

Comment:
@jamieomaguire Do we have a convention of starting private state variables with an underscore? Might be one for _the list_, if so?

Reply:
I think we sometimes do 😂 - adding to the list!